PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -428,70 +284,8 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
         dEsigmaDown=Var("userFloat('ecalTrkEnergyPostCorrNew')-userFloat('energySigmaDownNew')", float,  doc="ecal energy smearing value shifted 1 sigma up", precision=8),
 
 )
-##Keeping the possibilty of using V1 working points in older eras
```

Comment:
All the v1 cutbased and MVA IDs userFloats and userInts can now be removed as well from the `slimmedElectronsWithUserData` above

Reply:
noted